pixeltable/share/packager.py

```diff
@@ -225,9 +210,137 @@ def __build_tarball(self) -> Path:
         with tarfile.open(bundle_path, 'w:bz2') as tf:
             # Add metadata json
             tf.add(self.tmp_dir / 'metadata.json', arcname='metadata.json')
-            # Add the Iceberg warehouse dir (including the catalog)
-            tf.add(self.tmp_dir / 'warehouse', arcname='warehouse', recursive=True)
+            # Add the dir containing Parquet tables
+            tf.add(self.tables_dir, arcname='tables')
             # Add the media files
             for src_file, dest_name in self.media_files.items():
                 tf.add(src_file, arcname=f'media/{dest_name}')
         return bundle_path
+
+
+class TableRestorer:
+    tbl_path: str
```

Comment:
comment missing (what kind of path?)

Also: class comment missing

Reply:
done